Ginger/GingerCore/Platforms/PlatformsInfo/PlatformInfoBase.cs

```diff
@@ -75,6 +75,81 @@ public static PlatformInfoBase GetPlatformImpl(ePlatformType Platform)
                     return null;
             }
         }
+
+        /// <summary>
+        /// This method is used to check if the paltform supports recording
+        /// </summary>
+        /// <param name="Platform"></param>
+        /// <returns></returns>
+        public static bool IsPlatformSupportRecording(ePlatformType Platform)
```

Comment:
IsPlatformSupportRecording() should be removed, supporting recording or not should come by checking if (Agent.Driver is IRecord)

Reply:
Done